### ISSUE TITLE
SA1106 false positive on class with no body

### ISSUE THREAD

**Issue by CruseCtrl** [DotNetAnalyzers/StyleCopAnalyzers#3837]
To reproduce:

```cs
public class A;
```

This incorrectly triggers [SA1106](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1106.md) "The C# code contains an extra semicolon."

It feels like a similar thing to #3267, but that was for records and this is for classes

**Comment by bjornhellander** [comment]
I can't reproduct this with the latest beta. The issue you refer to was about a record, but the fix was not specific to records. Looks like you are simply using an old release.

**Comment by CruseCtrl** [comment]
Although it looks like the fix #3268 was merged nearly 4 years ago and version `1.1.118` was released on 29/04/2019. Is there going to be a new non-beta version any time soon?

**Comment by bjornhellander** [comment]
Not likely. See for example #3500.

**Comment by sharwell** [comment]
Duplicate of #3267
